File: 2998-count-symmetric-integers/count-symmetric-integers.py
class Solution:
    def countSymmetricIntegers(self, low: int, high: int) -> int:
        count = 0

        for i in range(low, high + 1):
            a = str(i)

            if(len(a) % 2 == 0):
                b = (len(a)//2)
                c = [int(i) for i in (a[:b])]
                d = [int(i) for i in (a[b:])]
                if(len(c) == 0):
                    pass
                elif(sum(c) == sum(d)):
                    count += 1
        
        return count
                
        

File: 2998-count-symmetric-integers/count-symmetric-integers.js
/**
 * @param {number} low
 * @param {number} high
 * @return {number}
 */
var countSymmetricIntegers = function (low, high) {
    let res = 0;

    for (let a = low; a <= high; ++a) {
        if (a < 100 && a % 11 === 0) {
            res++;
        } else if (1000 <= a && a < 10000) {
            const left = Math.floor(a / 1000) + Math.floor((a % 1000) / 100);
            const right = Math.floor((a % 100) / 10) + (a % 10);

            if (left === right) {
                res++;
            }
        }
    }

    return res;
};